webapp/src/app/pages/components/knowledge-graph/cytoscape/cytoscape.component.ts

```
@@ -320,13 +347,17 @@
               .style('display', 'none');
           }
           let nodesToHide = nodes.filter(function (e: any) {
-            return e.data.type === 'related_concept' || e.data.type === 'category';
+            return (
+              e.data.type === 'related_concept' || e.data.type === 'category'
+            );
           });
           for (var i = 0; i < nodesToHide.length; i++) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)